### DISCUSSION TITLE
Livewire doesn't catch ModelNotFoundException when findOrFail)=

### DISCUSSION THREAD

**Discussion by cscazorla** [livewire/livewire#8100] [7. Bugs (archived)]
### Livewire version

v3.4.4

### Laravel version

v10.43

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Try to fetch a model that doesn't exist in the DB with `findOrFail()`
2. Livewire doesn't catch ModelNotFoundException so you get the error: `Undefined property: Livewire\Features\SupportRedirects\Redirector::$headers`
3. If I wrap the findOrFail() instruction within a try/catch block, the exception is caught

### Repository or Wirebox URL

N/A

### Additional information

I found a similar issue:

https://github.com/livewire/livewire/issues/1805

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@cscazorla thanks for reporting! Any chance you could submit a failing test PR that covers this problem? Thanks!
